src/style.css

```diff
@@ -1 +1,131 @@
-/* empty source */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 14px;
+
+  --accent-color-blue: #00acdc;
+  --text-primary-color: #060b35;
+  --text-secondary-color: #616070;
+
+  color: var(--text-secondary-color);
+}
+
+body {
+  margin: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  color: var(--text-primary-color);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  color: var(--text-primary-color);
+  font-size: 10px;
+}
+
+.card__prices {
+  line-height: 18px;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  color: var(--text-primary-color);
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+
+  text-decoration: none;
+  text-transform: uppercase;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  background: #00acdc;
+  border-radius: 5px;
+  color: #fff;
```

Comment:
you also have this color used multiple times, consider creating variable for that